extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -13,3 +13,46 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+amp-sidebar {
+  color: black;
+  background-color: #efefef;
+  height: 100vh;
+  top: 0;
+  max-height: 100vh !important;
+  max-width: 80vw !important;
+  min-width: 45px !important;
+  outline: none;
+  z-index: 2147483647;
+}
+
+amp-sidebar::part(c),
+amp-sidebar::part(sidebar) {
+  /* Inherit values specified on amp-sidebar */
+  color: inherit;
+  background-color: inherit;
+  height: inherit;
+  top: inherit;
+  max-height: inherit;
+  max-width: inherit;
+  min-width: inherit;
+  outline: inherit;
+  z-index: inherit;
+
+  /* Use unspecified default values from amp-sidebar element */
+  width: inherit;
+  padding: inherit;
+  border: inherit;
+
+  /* Children of amp-sidebar should be visible */
+  visibility: visible !important;
```

Comment:
I think I understand why this is needed here, but can we limit this to the `amp-sidebar::part(c)` only? Setting it on the `amp-sidebar::part(sidebar)` will likely interfere with the component sooner or later.

Reply:
Good suggestion, I have updated in the latest revision.